nix/workbench/backend/nomad/cloud.sh

```diff
@@ -366,7 +366,7 @@ allocate-run-nomadcloud() {
     local curl_response
     # Makes `curl` return two objects, one with the body the other with
     # the headers, separated by a newline (`jq -s`).
-    if curl_response=$(curl --silent --show-error --write-out '%{json}' https://api.github.com/repos/input-output-hk/cardano-node/commits/"${gitrev}")
+    if curl_response=$(curl --silent --show-error --write-out '%{json}' https://api.github.com/repos/intersectMBO/cardano-node/commits/"${gitrev}")
```

Comment:
I don't think it matters but I would either use `IntersectMBO` or `intersectmbo` but not this mix

Reply:
Will adjust.